### PR TITLE
Feat: Added navigation

### PR DESCRIPTION
**Pull request by zoef-yt**
Added Navigation component to the CL.
image

### CODE REVIEW

File: pages/components/navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
+.navbar {
```
Comment (iRohitGaur) [inline pages/components/navigation/navigation.css:1]: UI looks really nice

File: pages/components/navigation/navigation.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+}
+
+.navbar-brand {
+	max-width: 30vw;
+	width: 100%;
+
+	font-size: calc(2.2rem + 0.9vw);
```
Comment (iRohitGaur) [inline pages/components/navigation/navigation.css:12]: space given here for grouping?

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

File: pages/components/navigation/navigation.css (hunk 2 above)
Comment (zoef-yt) [reply pages/components/navigation/navigation.css:12]: Yes.

**Comment by itsmebipul00** [comment]
UI looks awsome. Good to merge!

**Review by itsmebipul00 — APPROVED** [review]

**Pull request merged**